src/neta/neta.h

```diff
@@ -19,6 +20,13 @@ class NETADefinition
     NETADefinition(const SpeciesAtom *i, int maxDepth = 1);
     ~NETADefinition() = default;
 
+    public:
+    enum NETAFlags
```

Comment:
Bit more of an explicit name, since they are only relevant to creation.
```suggestion
    // NETA Definition Creation Flags
    enum NETACreationFlags
```

Reply:
The (now) `ExplicitHydrogens` flag is relevant to matching and the `IncludeRootElement` flag is relevant to creation, so I'll create two separate enums.